deps/exokit-bindings/videocontext/include/VideoMode.h

```diff
@@ -4,13 +4,21 @@
 #include "VideoCommon.h"
 
 namespace ffmpeg {
+class IVideoDevice {
```

Comment:
I don't understand the purpose of this interface. It seems there is only one kind of video device, and we could get by with only one `VideoDevce` and no vtables.